lib/modules/manager/woodpecker/index.ts

```diff
@@ -7,7 +7,7 @@ export const language: ProgrammingLanguage = 'docker';
 export { extractPackageFile };
 
 export const defaultConfig = {
-  fileMatch: ['(^|\\/)\\.woodpecker[^/]*\\.ya?ml$'],
+  fileMatch: ['(^|\\/)\\.woodpecker(?:\\/[^/]+)?\\.ya?ml$'],
```

Comment:
```suggestion
  fileMatch: ['^\\.woodpecker(?:\\/[^/]+)?\\.ya?ml$'],
```
should be better

/cc @MarcelCoding

Reply:
no, not supported by woodpecker by default. see docs link in PR description